style/PageWithMenu.mcss

```diff
@@ -122,3 +122,15 @@
   composes: fa-spin    from 'font-awesome/css/font-awesome.css';
   margin-right: 5px;
 }
+
```

Comment:
for the attach and detach icons we're using http://fontawesome.io/icons/ forward and reply, an attached volume in the to-be-created status view will have use the paperclip icon